src/index.js

```diff
@@ -52,4 +52,13 @@ process.on('SIGINT', cancelHandler)
 process.on('SIGTERM', cancelHandler)
 
 // Main
-main().then(() => require('./pre'))
+// Boolean inputs are stringified, but manually convert this input just in case that changes in the future.
+const emitTelemetry = core.getInput("emit_telemetry").toString();
+if emitTelemetry === "true" {
+  require('./pre')
+} else if (emitTelemetry === "false") {
+  main()
+} else {
+  // If emit_telemetry is not set, that indicates an older version of the dynamic workflow that doesn't separate telemetry from deployment
+  main().then(() => require('./pre'))
```

Comment:
This is to accomodate the time between tagging this release as `v1` and switching over to the new version of the dynamic workflow. We should expect to remove this last condition after both this action and the dynamic workflow on dotcom are deployed and in use.

Reply:
Just a note that when we revise this in that future PR, we should also either:
- set the default value of the `emit_telemetry` input to `"false"`, and/or
- reduce this section of the code to just use `else { ...` rather than `else if (emitTelemetry === "false")`.